tsconfig.json

```diff
@@ -15,6 +15,7 @@
     "strict": true,
     "noEmit": true,
     "baseUrl": ".",
+    "skipLibCheck": true,
```

Comment:
Don't want that here. This can easily hide real issues.

I've investigated the error and it looks like it has to be fixed in `@nuxt/types`